pkg/server/routes.go

```diff
@@ -90,10 +90,13 @@ func (s *Server) routes(
 	oktaConfig := s.NewOktaClientConfig()
 	jwtVerifier := okta.NewJwtVerifier(oktaConfig.OktaClientID, oktaConfig.OktaIssuer)
 
+	isLocalOrTestEnvironment := s.environment.Local() || s.environment.Test()
+
 	oktaMiddlewareFactory := okta.NewMiddlewareFactory(
 		handlers.NewHandlerBase(s.logger),
 		jwtVerifier,
 		store,
+		isLocalOrTestEnvironment,
```

Comment:
I am thinking that using `!s.environment.Prod()` here might be more succinct because we probably want to check these `_NONPROD` codes in the case of `s.environment.Impl()` as well

Reply:
Updated 👍